CONTRIBUTING.md

```diff
@@ -40,177 +264,782 @@ since someone else can often do this if you include a complete example. We
+someone fixed something in a commit initially authored by someone
+else), this should be reflected by adding ["Co-authored-by:"
+tags][GitHub-co-authored-by] at the end of the commit message.  The
+line should contain the co-author name and committer e-mail address.
+
+## Becoming a maintainer ##
+
+### Reviewing pull requests ###
+
+You can start reviewing PRs as soon as you feel comfortable doing so.
+Reviewers should ensure that the code that is changed or introduced is
+in good-shape and will not be a burden to maintain, is unlikely to
+break anything, or the compatibility-breakage has been identified and
+validated, includes documentation, changelog entries, and test files
+when necessary.  Reviewers can use labels, or change requests to
+further emphasize what remains to be changed before they can approve
+the PR.  Once reviewers are satisfied (regarding the part they
+reviewed), they should formally approve the PR, possibly stating what
+they reviewed.
+
+That being said, reviewers should also make sure that they do not make
+the contributing process harder than necessary: they should make it
+clear which comments are really required to perform before approving,
+and which are just suggestions.  They should strive to reduce the
+number of rounds of feedback that are needed by posting most of their
+comments at the same time.  If they are opposed to the change, they
+should clearly say so from the beginning to avoid the contributor
+spending time in vain.
+
+### Merging pull requests ###
+
+Our [CODEOWNERS][] file associates a team of maintainers, or a
+principal and secondary maintainers, to each component.  They will be
+responsible for self-assigning and merging PRs (they didn't co-author)
+that change this component.  When several components are changed in
+significant ways, at least a maintainer (other than the PR author)
+must approve the PR for each affected component before it can be
+merged, and one of them has to assign the PR, and merge it when it is
+time.  Before merging, the assignee must also select a milestone for
+the PR (see also Sec. [Release Management](#release-management)).
+
+If you feel knowledgeable enough to maintain a component, and have a
+good track record of contributing to it, we would be happy to have you
+join one of the maintainer teams.
+
+The merging process is described in more details in [this
+document][MERGING].
+
+The people with merging powers (either because listed as a principal
+or secondary maintainer in [CODEOWNERS][], or because member of a
+maintainer team), are the members of the **@coq/pushers** team.
+
+### Core development team ###
+
+The core developers are the active developers with a lengthy and
+significant contribution track record.  They are the ones with admin
+powers over the Coq organization, and the ones who take part in votes
+in case of conflicts to take a decision (rare).  One of them is
+designated as a development coordinator, and has to approve the
```

Comment:
Who is that?

Reply:
Matthieu. Isn't it how we call our GRAND LEADER?